dev/tests/static/framework/Magento/Sniffs/Annotation/MethodArgumentsSniff.php

```diff
@@ -124,8 +123,7 @@ private function getMethodArguments(File $phpcsFile, int $openParenthesisPtr, in
     private function getMethodParameters(array $paramDefinitions): array
     {
         $paramName = [];
-        $paramCount = count($paramDefinitions);
-        for ($i = 0; $i < $paramCount; $i++) {
+        foreach (array_keys($paramDefinitions) as $i) {
```

Comment:
`array_keys` is not a good idea. Why cannot use previous implementation? If there are gaps in keys, use `foreach` and ignore values.

Reply:
Same for all other changed loops.